ansible/roles/service/hippo_authoring/files/hippo_authoring.yml

```diff
@@ -0,0 +1,280 @@
+    ConstraintDescription: must be the name of an existing EC2 KeyPair.
+
+  # SSLCertificateId:
+  #   Description: SSL Certificate ARN
+  #   Type: String
+
+Resources:
+  SG:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Enable acces from VPC
+      VpcId:
+        Fn::ImportValue: !Sub "${Environment}-vpc::VpcId"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort:  22
+          ToPort:  22
+          CidrIp:
+            Fn::ImportValue: !Sub "${Environment}-vpc::CidrBlock"
+        - IpProtocol: tcp
+          FromPort:  8080
+          ToPort:  8080
+          CidrIp:
+            Fn::ImportValue: !Sub "${Environment}-vpc::CidrBlock"
+      Tags:
+        - Key: Name
+          Value: !Sub "${Environment}-hippo-authoring"
+
+  LaunchConfig:
+    Type: AWS::AutoScaling::LaunchConfiguration
+    Metadata:
+      AWS::CloudFormation::Init:
+        config:
+          files:
+            "/bootstrap/environment.yml":
+              content: !Ref EnvironmentConfig
+              group: root
+              mode: "000644"
+              owner: root
+    Properties:
+      AssociatePublicIpAddress: false
+      ImageId: !Ref Image
+      InstanceMonitoring: !Ref InstanceMonitoring
+      InstanceType: !Ref InstanceType
+      KeyName: !Ref KeyName
+      SecurityGroups:
+        - Ref: SG
+      UserData:
+        Fn::Base64: !Sub |
+          #!/bin/bash
+          export HOME=/root
+          . /bootstrap/bootstrap.inc.sh
+          set_global 'LaunchConfig' 'ASGroup' '${AWS::Region}' '${AWS::StackName}'
+          cfnInit
+          cd /bootstrap && make ansible_configure_local ROLE=hippo_authoring \
+            || die "configure playbook failed"
+          log "Instance bootstrapped"
+          signal 0
+          cleanup
+
+  ASGroup:
+    Type: AWS::AutoScaling::AutoScalingGroup
+    Properties:
+      DesiredCapacity: !Ref InstanceCount
+      LaunchConfigurationName: !Ref LaunchConfig
+      LoadBalancerNames:
+        - !Ref LoadBalancer
+      MaxSize: 6
+      MinSize: !Ref InstanceCount
+      Tags:
+        - Key: Name
+          Value: !Sub "${Environment}-hippo-authoring"
+          PropagateAtLaunch: true
+      VPCZoneIdentifier:
+        Fn::Split:
+          - ","
+          - Fn::ImportValue: !Sub "${Environment}-vpc::SubnetsPrivate"
+    CreationPolicy:
+      ResourceSignal:
+        Count: 0
+        Timeout: PT10M
+    UpdatePolicy:
+      AutoScalingRollingUpdate:
+        MinInstancesInService: 1
+        MaxBatchSize: 1
+        PauseTime: PT10M
+        WaitOnResourceSignals: true
+
+  ##
+  # Loadbalancing
+  #
+  LoadBalancerSg:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Enable acces from VPC
+      VpcId:
+        Fn::ImportValue: !Sub "${Environment}-vpc::VpcId"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 156.67.242.53/32
```

Comment:
Shall we have this as a parameter?

Reply:
This is tmp solution anyway. The next version will allow us to have more dynamic list of source IPs